docs/pages/identity-governance/aws-iam-identity-center/guide.mdx

```diff
@@ -113,6 +115,21 @@ the same [IdP](../../zero-trust-access/sso/sso.mdx) as your current Identity Cen
 external identity source.
 </Admonition>
 
+<Admonition type="info" title="SAML IdP RBAC">
+In this step, Teleport creates a `saml_idp_service_provider` resource for the 
+AWS IAM Identity Center. 
+
+Starting with Teleport version 18 and role version 8, the SAML IdP service provider 
+resources are gated behind `app_labels` label matchers. If you are using Teleport 
+role version 8 to manage access, you must update the role to grant access to this 
+service provider resource with `app_labels` matching `teleport.dev/origin: aws-identity-center` label.
+```yaml
+  allow:
+    app_labels:
+      'teleport.dev/origin': 'aws-identity-center'
```

Comment:
This is pretty unintuitive, and definitely should be a part of the guide, not just a note. In addition, I think we should update the [custom role example](https://goteleport.com/docs/identity-governance/aws-iam-identity-center/guide/#creating-custom-identity-center-roles) to use role v8 and set proper app label as well.

Reply:
Also, we don't have to do it now but I wonder if we should have a preset "aws-identity-center" role that includes this app label.

Reply:
Agree. I do not like this either. Also suggesting origin label for RBAC does not sit right with me. Long term, I plan to add a new label when the SAML app is created for the identity center and reference that here. Ideally, create a "preset" kind of role and reference that.  @smallinsky and I have been discussing similar ideas to simplify this. 

Good point on updating [custom role example](https://goteleport.com/docs/identity-governance/aws-iam-identity-center/guide/#creating-custom-identity-center-roles) instead. I will do that.

Reply:
I have updated this in a new commit: https://github.com/gravitational/teleport/pull/58667/commits/372ad0181c88c7d99129bf86c8439795af42aa47
- The app label reference is added to the pre-requisite and in the [custom role example](https://goteleport.com/docs/identity-governance/aws-iam-identity-center/guide/#creating-custom-identity-center-roles)
- Remove admonition but kept a short description for the step to mention saml service provider is being configured